wordpress-mobile/gutenberg-mobile
# test: Improve E2E test consistency

Avoid setup scripts during test runs to mirror changes in
https://github.com/WordPress/gutenberg/pull/55554.

To test: 

- Verify the `npm run core test:e2e:setup` command succeeds when run.
- Verify the `npm run test:e2e:ios:local` command does not throw errors.
- Verify the `npm run test:e2e:android:local` command does not throw errors.

PR submission checklist:
